extension/src/lib/extensionHolder.js

```diff
@@ -0,0 +1,58 @@
+/*globals fluid */
+
+"use strict";
+
+var gpii = fluid.registerNamespace("gpii");
+var chrome = chrome || require("sinon-chrome");
+
+fluid.defaults("gpii.chrome.extensionHolder", {
+    gradeNames: "fluid.modelComponent",
+    extensionId: "",
+    members: {
+        extensionInstance: null
+    },
+    model: {
+        extensionEnabled: undefined
+    },
+    invokers: {
+        setEnabled: {
+            funcName: "gpii.chrome.extensionHolder.setEnabled",
+            args: ["{that}", "{arguments}.1"]
+        },
+        getEnabled: {
+            funcName: "gpii.chrome.extensionHolder.getEnabled",
+            args: "{that}.extensionInstance"
+        }
```

Comment:
Yes, you can listen to events from extensions/apps (onEnabled, onInstalled, etc), but in this particular case I'm just performing a call to [get](https://developer.chrome.com/extensions/management#method-get), which returns an [ExtensionInfo](https://developer.chrome.com/extensions/management#type-ExtensionInfo) object (what I call extensionInstance, bad naming btw ...), but you can't listen to events from it. We can listen to events in general, and we can make some filtering to listen for events from a particular extension/app. It can probably be helpful for testing, but this is something I still need to deep dive into.